frontend/src/Dashboard/actions.async.unit.spec.js

```diff
@@ -0,0 +1,64 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from './actions';
+import fetchMock from 'fetch-mock';
+import {expect} from 'chai';
+
+const middleware = [thunk];
+const mockStore = configureMockStore( middleware);
+
+const expectedHouseIssues = [
+  {
+    title: "Title",
+    location: {
+      street: "street",
+      postCode: "post code"
+    },
+    owner: "user_mongo_id",
+    images: [],
+    issues: []
+  }
+];
+
+describe( 'async Dashboard actions', () => {
+  afterEach(() => {
+    fetchMock.reset();
+    fetchMock.restore();
+  });
+  it( 'should create actions when fetching house issues', () => {
+    fetchMock.getOnce('/api/v1/house-issues', {
+      body: { houseIssues: expectedHouseIssues },
+      headers: {'content-type': 'application/json'}
+    });
+    const expectedActions = [
+      { type: actions.HOUSE_ISSUES_HAS_ERRORED, hasErrored: false },
```

Comment:
I would suggest you to use action creators, always try to have a single source of knowledge.